core/src/main/scala/cats/data/Const.scala

```diff
@@ -161,10 +171,26 @@ sealed abstract private[data] class ConstInstances extends ConstInstances0 {
       ): Eval[C] =
         f(fab.getConst, c)
     }
+
+  implicit def catsDataMonoidKForConst[C: Monoid]: MonoidK[Const[C, *]] = new MonoidK[Const[C, *]] {
+    override def empty[A]: Const[C, A] = Const.empty
+    override def combineK[A](x: Const[C, A], y: Const[C, A]): Const[C, A] = x.combine(y)
+  }
+
+  implicit def catsDataSemigroupKForConst[C: Semigroup]: SemigroupK[Const[C, *]] = new SemigroupK[Const[C, *]] {
+    override def combineK[A](x: Const[C, A], y: Const[C, A]): Const[C, A] = x.combine(y)
+  }
```

Comment:
Just to make sure we are not missing it: wouldn't the following work too?
```suggestion
  implicit def catsDataMonoidKForConst[C: Semigroup]: MonoidK[Const[C, *]] = new MonoidK[Const[C, *]] {
    override def empty[A]: Const[C, A] = Const.empty
    override def combineK[A](x: Const[C, A], y: Const[C, A]): Const[C, A] = x.combine(y)
  }
```
I.e. it looks like even for `MonoidK` it is just `C: Semigroup` that is in use. So since MonoidK <: SemigroupK, wouldn't be it possible to have just one instance of `MonoidK` that demands `C: Semigroup` instead of two separate ones?

Reply:
I think you need `Monoid` for `Const.empty` because `Const` always has an element inside.

Reply:
Ah yes, you're right. I overlooked it somehow. Thank you for the clarification.